provisioning/device/test/_polling_state_machine_test.js

```diff
@@ -42,6 +42,21 @@ var operationStatusReturnsAssigningThenAssigned = function () {
   });
 };
 
+var registrationRequestReturnsRegistering = function () {
+  var callCount = 0;
```

Comment:
>callCount [](start = 6, length = 9)

a sinon spy has a callCount property. no need to maintain our own.

Reply:
Please show how to using this internal to this test.  This is not clear.

---
In reply to: [282193298](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282193298) [](ancestors = 282193298)

Reply:
var stub = sinon.stub().callsFake(function () { console.log('call count: ' + stub.callCount); })

---
In reply to: [282194966](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282194966) [](ancestors = 282194966,282193298)

Reply:
I should have googled it.  Sorry.

---
In reply to: [282209932](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282209932) [](ancestors = 282209932,282194966,282193298)